scalafix-tests/input/src/main/scala-2-xsource3/test/organizeImports/TargetDialectAuto.scala

```diff
@@ -0,0 +1,13 @@
+/*
+rules = [OrganizeImports]
+OrganizeImports {
+  removeUnused = false
+  targetDialect = Auto
```

Comment:
testing `-Xsource:3` detection thanks to https://github.com/scalacenter/scalafix/pull/1666